leases/lease.go

```diff
@@ -68,7 +68,12 @@ func SynchronousDelete(ctx context.Context, o *DeleteOptions) error {
 // WithLabels sets labels on a lease
 func WithLabels(labels map[string]string) Opt {
 	return func(l *Lease) error {
-		l.Labels = labels
```

Comment:
Currently, `WithLabels` comment uses `set`. I think we should also update the comment, like `merge into` or something?

Reply:
Great point. `Snapshots.WithLabels` uses "appends" but I think "merges" may be more appropriate in this case.